pytorch_lightning/trainer/trainer.py

```diff
@@ -462,9 +462,6 @@ def __init__(
         )
         fit_loop = FitLoop(min_epochs=min_epochs, max_epochs=max_epochs)
         training_epoch_loop = TrainingEpochLoop(min_steps=min_steps, max_steps=max_steps)
-        training_batch_loop = TrainingBatchLoop()
-        training_validation_loop = EvaluationLoop()
-        training_epoch_loop.connect(batch_loop=training_batch_loop, val_loop=training_validation_loop)
```

Comment:
this is unrelated?

Reply:
It's not, this was unsetting the `training_validation_loop` value that the `training_epoch_loop` sets. Setting these is not necessary.

Another option would have been to also add `verbose=False` here